scalikejdbc-mapper-generator/src/sbt-test/scalikejdbc-mapper-generator/gen/src/test/scala/ProgrammersTest.scala

```diff
@@ -38,6 +38,9 @@ class ProgrammersTest extends fixture.FlatSpec with Matchers with AutoRollback {
     programmers.foreach(Programmers.destroy(_))
     Programmers.findAll().toSet should equal(Set.empty[Programmers])
     Programmers.countAll() should equal(0)
+
+    val res = Programmers.batchInsert(programmers.toList)
```

Comment:
@seratch 

if I remove toList, it will fails..
toSeq or toList are reasonable here I think. 

around on line 18, `val programmers` start as Set by using to 'toSet' method 
```scala
val programmers = List(Some("aaa"), Some("bbb"), None).map(name =>
  Programmers.create(name = name, t1 = new DateTime(2014, 12, 31, 20, 0))
     ).toSet	
```

line 18's `toSet` is necessary to tset findAll method for non order collection. (@xuwei-k told me , and I missed it..)
So, if I want to use this `programmers: Set[Programmers]` for batchInsert, I have to change it toSeq or toList on line 42...